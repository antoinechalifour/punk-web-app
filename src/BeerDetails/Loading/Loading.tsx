import React from "react";
import { TextBlock, RoundShape } from "react-placeholder/lib/placeholders";

import { Template } from "../Template";

export interface LoadingProps {}

export const Loading: React.FunctionComponent<LoadingProps> = () => (
  <Template
    brewersTips={<TextBlock rows={3} color="rgba(255, 255, 255, .2)" />}
    description={<TextBlock rows={4} color="rgba(0, 0, 0, .2)" />}
    generalInformation={<TextBlock rows={2} color="rgba(0, 0, 0, .2)" />}
    header={
      <div>
        <TextBlock rows={2} color="rgba(255, 255, 255, .2)" />
      </div>
    }
    hops={<TextBlock rows={5} color="rgba(0, 0, 0, .2)" />}
    image={
      <RoundShape
        color="#f5f5f5"
        style={{
          height: 150,
          width: 150,
          margin: "auto",
          boxShadow:
            "0 1px 3px rgba(0, 0, 0, 0.1), 0 5px 10px rgba(84, 145, 247, 0.2)"
        }}
      />
    }
    malts={<TextBlock rows={3} color="rgba(0, 0, 0, .2)" />}
    yeast={<TextBlock rows={1} color="rgba(0, 0, 0, .2)" />}
  />
);
