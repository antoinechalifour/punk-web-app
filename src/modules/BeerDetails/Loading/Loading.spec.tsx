import { shallow } from "enzyme";
import React from "react";

import { Loading, LoadingProps } from "./Loading";

it("should match snapshot", () => {
  const props: LoadingProps = {};

  const wrapper = shallow(<Loading {...props} />);

  expect(wrapper).toMatchSnapshot();
});
