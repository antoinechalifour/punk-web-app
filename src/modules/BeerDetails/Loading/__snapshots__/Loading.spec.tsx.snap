// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot 1`] = `
<Template
  brewersTips={
    <TextBlock
      color="rgba(255, 255, 255, .2)"
      rows={3}
      widths={
        Array [
          97,
          100,
          94,
          90,
          98,
          95,
          98,
          40,
        ]
      }
    />
  }
  description={
    <TextBlock
      color="rgba(0, 0, 0, .1)"
      rows={4}
      widths={
        Array [
          97,
          100,
          94,
          90,
          98,
          95,
          98,
          40,
        ]
      }
    />
  }
  foodPairing={
    <TextBlock
      color="rgba(0, 0, 0, .1)"
      rows={2}
      widths={
        Array [
          97,
          100,
          94,
          90,
          98,
          95,
          98,
          40,
        ]
      }
    />
  }
  generalInformation={
    <TextBlock
      color="rgba(0, 0, 0, .1)"
      rows={2}
      widths={
        Array [
          97,
          100,
          94,
          90,
          98,
          95,
          98,
          40,
        ]
      }
    />
  }
  header={
    <div>
      <TextBlock
        color="rgba(255, 255, 255, .2)"
        rows={2}
        widths={
          Array [
            97,
            100,
            94,
            90,
            98,
            95,
            98,
            40,
          ]
        }
      />
    </div>
  }
  hops={
    <TextBlock
      color="rgba(0, 0, 0, .1)"
      rows={5}
      widths={
        Array [
          97,
          100,
          94,
          90,
          98,
          95,
          98,
          40,
        ]
      }
    />
  }
  image={
    <RoundShape
      color="#f5f5f5"
      style={
        Object {
          "boxShadow": "0 1px 3px rgba(0, 0, 0, 0.1), 0 5px 10px rgba(84, 145, 247, 0.2)",
          "height": 150,
          "margin": "auto",
          "width": 150,
        }
      }
    />
  }
  malts={
    <TextBlock
      color="rgba(0, 0, 0, .1)"
      rows={3}
      widths={
        Array [
          97,
          100,
          94,
          90,
          98,
          95,
          98,
          40,
        ]
      }
    />
  }
  yeast={
    <TextBlock
      color="rgba(0, 0, 0, .1)"
      rows={1}
      widths={
        Array [
          97,
          100,
          94,
          90,
          98,
          95,
          98,
          40,
        ]
      }
    />
  }
/>
`;
